.travis.yml

```diff
@@ -1,31 +1,33 @@
 language: python
 sudo: false
+cache: pip
+git:
+   depth: 1
 python:
 - 2.6
 - 2.7
-- 3.2
```

Comment:
I think it's not worth now.  Could you remove other codes related Python 3.2 support (e.g. tox.ini, classifiers in setup.py metadata, README.rst, docs/index.rst) as well?